app/uk/gov/hmrc/eoricommoncomponent/frontend/views/match_organisation_utr.scala.html

```diff
@@ -47,8 +47,8 @@
 
 @headerAndTitle = @{
     organisationType match {
-        case CdsOrganisationType.ThirdCountryOrganisationId => messages("cds.matching.row-organisation.utr.title-and-heading")
-        case CdsOrganisationType.ThirdCountrySoleTraderId | CdsOrganisationType.ThirdCountryIndividualId =>
+        case CdsOrganisationType.ThirdCountryOrganisationId | CdsOrganisationType.IsleOfManOrganisationId => messages("cds.matching.row-organisation.utr.title-and-heading")
+        case CdsOrganisationType.ThirdCountrySoleTraderId | CdsOrganisationType.ThirdCountryIndividualId  | CdsOrganisationType.IsleOfManSoleTraderId | CdsOrganisationType.IsleOfManIndividualId =>
```

Comment:
you could clean up these by making this a set in CdsOrganisationType, or do a similar check to that described earlier.